share/p2p/peers/manager.go

```diff
@@ -390,15 +390,14 @@ func (m *Manager) getOrCreatePool(datahash string) *syncPool {
 }
 
 func (m *Manager) blacklistPeers(reason blacklistPeerReason, peerIDs ...peer.ID) {
-	log.Debugw("blacklisting peers",
-		"peers", peerIDs,
-		"reason", reason)
 	m.metrics.observeBlacklistPeers(reason, len(peerIDs))
 
-	if !m.params.EnableBlackListing {
-		return
-	}
 	for _, peerID := range peerIDs {
+		log.Debugw("blacklisting peer", "peer", peerID.String(), "reason", reason)
+		if !m.params.EnableBlackListing {
+			continue
+		}
```

Comment:
can you put debug log after this then?

Reply:
I made it before, to make it possible to debug blacklisting while it is disabled.

Reply:
can you make a note of that then 😬  hopefully we'll remove this EnableBlacklisting altogether soon

Reply:
I've added log line, to clarify why it is always logged